test/uk/gov/hmrc/merchandiseinbaggagefrontend/pagespecs/pages/CannotUseServicePage.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.merchandiseinbaggagefrontend.pagespecs.pages
+
+import org.openqa.selenium.WebDriver
+
+class CannotUseServicePage(baseUrl: BaseUrl)(implicit webDriver: WebDriver) extends BasePage(baseUrl) {
+  override val path: String = CannotUseServicePage.path
```

Comment:
I don't like having auth on the object and again on the class, will refactor this.